src/Engine/Scene/Parsing/SceneLoader.cpp

```
@@ -72,6 +72,39 @@ void SceneLoader::loadDeserialBindings() {
+                    orbitalElems.parentBody = ctx->sceneEntities->at(YAMLScene::Body_Sun);
+                    m_ecsRegistry->addComponent(ctx->entityID, orbitalElems);
+                }
+                break;
+
+            case SSB:
+                // IMPORTANT: The actual central orbiting body of the simulation is NOT the Sun in SSB, but the Solar System Barycenter. Setting the parent body to the Sun is a temporary fix, and does not reflect physical reality.
+                // TODO: IMPLEMENT SOLAR SYSTEM BARYCENTER AS CENTRAL ORBITING BODY FOR SSB-FRAME SIMULATIONS
+
+                if (ctx->entityName != YAMLScene::Body_Sun) {
+                    orbitalElems.parentBody = ctx->sceneEntities->at(YAMLScene::Body_Sun);
+                    m_ecsRegistry->addComponent(ctx->entityID, orbitalElems);
```

Cppcheck (2.20.0)

[performance] Function 'getName()' should return member 'm_threadName' by const reference.

(returnByReference)